Error: Cannot find module '@actions/core'

node:internal/modules/cjs/loader:1215
  throw err;
  ^
Error: Cannot find module '@actions/core'
Require stack:
- /home/runner/work/_actions/PavanMudigondaTR/variable-substitution/v1.0/lib/variableSubstitution.js
    at Module._resolveFilename (node:internal/modules/cjs/loader:1212:15)
    at Module._load (node:internal/modules/cjs/loader:1043:27)
    at Module.require (node:internal/modules/cjs/loader:1298:19)
    at require (node:internal/modules/helpers:182:18)
    at Object.<anonymous> (/home/runner/work/_actions/PavanMudigondaTR/variable-substitution/v1.0/lib/variableSubstitution.js:12:14)
    at Module._compile (node:internal/modules/cjs/loader:1529:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1613:10)
    at Module.load (node:internal/modules/cjs/loader:12[75](https://github.com/tr/cicd_gh-actions-ado-testplans/actions/runs/14382561670/job/40329810272#step:3:79):32)
    at Module._load (node:internal/modules/cjs/loader:1096:12)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:164:12) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/home/runner/work/_actions/PavanMudigondaTR/variable-substitution/v1.0/lib/variableSubstitution.js'
  ]
}

--- a/lib/variableSubstitution.js
+++ b/lib/variableSubstitution.js
@@ -1,3 +1,4 @@
+const core = require('@actions/core');
 "use strict";
 var __awaiter = (this && this.__awaiter) || function (thisArg, _arguments, P, generator) {
     function adopt(value) { return value instanceof P ? value : new P(function (resolve) { resolve(value); }); }
@@ -12,7 +13,7 @@ Object.defineProperty(exports, "__esModule", { value: true });
 const core = require("@actions/core");
 const envVariableUtility_1 = require("./operations/envVariableUtility");
 const jsonVariableSubstitutionUtility_1 = require("./operations/jsonVariableSubstitutionUtility");
-const xmlDomUtility_1 = require("./operations/xmlDomUtility");
+const xmlDomUtility_1 = require("./xmlDomUtility");
 const xmlVariableSubstitution_1 = require("./operations/xmlVariableSubstitution");
 const utility_1 = require("./operations/utility");
 const fs = require("fs");

